pkg/embeddedetcd/config.go

```diff
@@ -98,7 +98,7 @@ func NewConfig(o options.CompletedOptions, enableWatchCache bool) (*Config, erro
 		if err != nil {
 			return nil, err
 		}
-		cfg.ListenMetricsUrls = append(cfg.LPUrls, *u)
+		cfg.ListenMetricsUrls = append(cfg.LPUrls, *u) // TODO: What was the intent here?
```

Comment:
@sttts What was your intent here in #1608 ? The `wastedassign` linter IMHO rightfully complains about this line.

Reply:
oh, now I got the question. Don't know. Can you git blame? @stevekuznetsov was that your code? Do you remember? Typo?

Reply:
https://github.com/kcp-dev/kcp/commit/2b9193adc750b7005b3b5108d8be3cc09365fae5#diff-aece0d70693f229fcbf4f1ca12cc1beec3066aa4059b0a054b924d790b9c8b23L107-L109

Reply:
`cfg.LPUrls` looks wrong, should be `cfg.ListenMetricsUrls`.